packages/application-components/src/components/modal-pages/form-modal-page/README.md

```diff
@@ -8,7 +8,7 @@ import { FormModalPage } from '@commercetools-frontend/application-components';
 
 ## Description
 
-Form Modal pages are controlled components used to render a page with a form or something that requires user input, similar to `InfoModalPage` but semantically with a different role. The header includes buttons to control the submission or cancellation of the form (or which can be overriden with other custom controls).
+Form Modal pages are controlled components used to render a page with a form or something that requires user input, using `CustomFormModalPage` with default controls.
```

Comment:
Maybe we should specify a bit more in detail when you would need to opt-out and use the custom component.